src/main/steps/common/page.njk

```diff
@@ -80,6 +80,10 @@
         {% endblock %}
     </div>
   </div>
+  <div class="govuk-grid-row">
```

Comment:
@rahulsanjava19 Can we try to reuse existing block if works instead of creating new one.
There is one block page_under_under_form, can we try if this works ?
`      {% block page_content_under_form %}
      {% endblock %}`